generators/entity-client/templates/react/src/main/webapp/app/entities/_entity.tsx

```diff
@@ -19,23 +19,34 @@
 import * as React from 'react';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
-import { Button } from 'reactstrap';
+import { Button<%_ if (searchEngine === 'elasticsearch') { _%>, InputGroup<%_ } _%>
+ } from 'reactstrap';
```

Comment:
put this in same line above
`import { Button<% if (searchEngine === 'elasticsearch') { %>, InputGroup<% } %> } from 'reactstrap';`